src/libAtomVM/term.h

```diff
@@ -1106,6 +1111,33 @@ static inline term term_create_empty_binary(size_t size, Heap *heap, GlobalConte
     return t;
 }
 
+static inline bool term_normalize_binary_pos_len(term binary, avm_int_t pos, avm_int_t len, BinaryPosLen *pos_len)
```

Comment:
Does this function have any purpose outside `binary:match` or is it just a helper function for it?